### DISCUSSION TITLE
Suggestions to reduce VRAM usage

### DISCUSSION THREAD

**Discussion by caniko** [hyprwm/Hyprland#4121] [Q&A]
I have a laptop with an iGPU that has `512Mi` of VRAM, which is often full. Hyprland is using about 365-420MiBs of VRAM, is there any way to lower this without lowering the resolution of my monitors?

**Note**
I have two hi-res monitors (3440x1440 and 2560x1600).

**Comment by vaxerski** [comment]
Nope, unless some optimizations are done to hyprland's vram usage, which I might consider some day. It's not _really_ that bad, so it's not a priority.
